weblate/machinery/microsoft.py

```diff
@@ -108,23 +117,24 @@ def download_languages(self):
         if isinstance(payload, str):
             raise Exception(payload)
 
-        return payload
+        return list(payload['translation'].keys())
```

Comment:
```suggestion
        return payload['translation'].keys()
```
Type casting is not needed here, the set is created from this in caller.